src/server/index.js

```diff
@@ -75,7 +75,9 @@ app.prepare()
         server.use(keycloakClient.middleware());
 
         logger.info("adding the /login handler");
-        server.get("/login", keycloakClient.protect());
+        server.get("/login", keycloakClient.protect(), (req, res) => {
+            res.redirect("/");
+        });
```

Comment:
Apparently we were never hitting the `/login` URL directly before. It must have always been `/login/dashboard` or something like that.